csrc/index_compute.cpp

```diff
@@ -1979,11 +1984,14 @@ std::vector<Val*> Index::getStrides(TensorView* tv) {
     for (const auto i : c10::irange(alloc_dom.size())) {
       if (alloc_dom[i]->isReduction() || alloc_dom[i]->isStride()) {
         strides[i] = GpuLower::current()->kernel()->oneVal();
-        continue;
+      } else if (alloc_dom[i]->isDeviceDim()) {
+        strides[i] = GpuLower::current()->kernel()->oneVal();
```

Comment:
Does this matter? Indexing of DID parallelized allocation domains should be always zero, right? If so, strides shouldn't matter.

Reply:
Actually, you are right there shouldn't be a special case for DeviceDim; it should be handled in the else branch. 
These were just bandaids to avoid the actual problem in expr_evaluator that Sam fixed in https://github.com/NVIDIA/Fuser/pull/1900. 
I will close this PR and clean it up.